### ISSUE TITLE
Not able to inspect react native drop-down element in iOS

### ISSUE THREAD

**Issue by Sanoop-PM** [appium/appium#20618]
Am trying to automate below drop down but Appium inspector is unable to detect the element. Even we tried with iOS Accessibility Inspector issue is same.

**Drop down:**
[react-native-element-dropdown - npm (npmjs.com)](https://www.npmjs.com/package/react-native-element-dropdown)

I tried below setting changes in Appium mentioned in the page ([Troubleshooting - Appium Inspector](https://appium.github.io/appium-inspector/latest/troubleshooting/#cannot-see-full-source-tree-with-xcuitest-driver-react-native-app)), but it didn’t help.

**Note:** For Android same component its working fine.

Could you please help to come over this issue?

**Appium Log:**
[Appium Log.txt](https://github.com/user-attachments/files/17145390/Appium.Log.txt)

**Appium Inspector:**

<img width="452" alt="image" src="https://github.com/user-attachments/assets/d6b4dc70-0a56-43c7-94de-fc4240341c0a">

**iOS Accessibility Inspector:**

<img width="452" alt="image" src="https://github.com/user-attachments/assets/6efd4964-3b18-4d2a-b7f5-70e2b948b40e">

I tried to debug the react application but in react am able to see its value is getting rendered.

<img width="1423" alt="image" src="https://github.com/user-attachments/assets/ebdbe5bf-c524-4949-bacc-90023667d2b5">

**Comment by mykola-mokhnach** [comment]
> Even we tried with iOS Accessibility Inspector issue is same.

There is no magic there. The driver simply reflects what the app under test exposes to the accessibility layer. consult react guidelines on how to make the above control more accessible. Unfortunately there is nothing we could help there from the driver side.

Closed as third party issue.
